linux/RNGEntropy.c

```diff
@@ -0,0 +1,83 @@
+/*
+htop - RNGEntropy.c
+(C) 2022 Loic Jourdheuil
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "linux/RNGEntropy.h"
+
+#include "CRT.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/statfs.h>
+#include <sys/statvfs.h>
+
+#include "Object.h"
+#include "XUtils.h"
+
+static void getRNGEntropy(Meter* this) {
+   FILE* file = fopen(PROCDIR "/sys/kernel/random/entropy_avail", "r");
```

Comment:
Not sure if going with that whole `fopen` API and its overhead to read just a single integer is worth the overhead.